hydra-node/json-schemas/api.yaml

```diff
@@ -234,6 +234,8 @@ components:
         required:
           - tag
           - me
+          - headStatus
+          - snapshotUtxo
```

Comment:
Must: Update the api spec to indicate this is optional.

Drop this from the `required` list, as it may not be included (unless something else comes out here https://github.com/input-output-hk/hydra/issues/823#issuecomment-1521750111)